website/project/spam/model.py

```diff
@@ -39,6 +39,7 @@ class SpamMixin(StoredObject):
     HAM = 4
 
     spam_status = fields.IntegerField(default=UNKNOWN, index=True)
+    latest_report = fields.DateTimeField(default=None, index=True)
```

Comment:
`latest_report` is a bit of a misnomer (it's not a report). For consistency with other datetime fields, how about `date_last_reported`?